src/drivers/imx/dummy-dma.c

```diff
@@ -3,38 +3,153 @@
 // Copyright 2019 NXP
```

Comment:
Subject should contain a verb. e.g drivers: imx: Add implementation for dummy DMA driver